.travis.yml

```diff
@@ -62,6 +74,13 @@ env:
     - list_id_Done=5ef1186dcbda554f16c6d66f
     - ANON_USER=TheLegendaryPan
     - ANON=TheLegendaryPanTester
+    # added as part of module 12
+    # - ARM_CLIENT_ID=27f4532f-cba5-4317-8666-9084d1bea341
+    # - ARM_TENANT_ID=7d6f97d6-d755-4c10-b763-409cc4b6638f
+    # - ARM_SUBSCRIPTION_ID=d33b95c7-af3c-4247-9661-aa96d47fccc0
+    # - ARM_CLIENT_SECRET=r6tBPNQ.Jwb3UspM.0LZrqY99ndiPWumd7
```

Comment:
This is a secret and definitely shouldn't be checked in to our code - again we should delete this and regenerate that! (Easiest might be to delete the existing Service Principal and then create a fresh one)